OpenTyping/KeyBox.xaml.cs

```diff
@@ -115,14 +147,38 @@ public void Release()
                 KeyBack.Height += PressDiff;
                 Canvas.SetTop(KeyBack, 0);
             }
-
+            
             KeyColor = defaultKeyColor;
             ShadowColor = defaultShadowColor;
             handPopup.IsOpen = false;
 
             Pressed = false;
         }
 
+        public void Release(bool pressed)
```

Comment:
pls give me the detail why this function overriding is needed.

Reply:
Because when we enable keyboard to be colorfull and use the default Release function, after pressing a key it is always white. That's why I removed just two lines of code and oveerload the function.

Reply:
In that case, the parameter of the function can be a defaultKeyColor rather than "pressed"